jumpscale/core/identity/__init__.py

```diff
@@ -89,6 +89,22 @@ def register(self, host=None):
 
         return user.id
 
+    def get_threebot_config(self):
+        return self._threebot_data
+
+    def add_admin(self, name):
+        config = j.core.config.get_config()
+        config["threebot"]["admins"].append(name)
```

Comment:
duplicate code i see
```
config = j.core.config.get_config()
config["threebot"]["admins"]
``` 
everywhere, maybe use at the top ?

Reply:
done